### PR TITLE
CE complete. Ready for SME.

### PR DESCRIPTION
**Pull request by VinayApposite**
Please check my comments.

### CODE REVIEW

File: Instructions/20480C_MOD05_LAB_MANUAL.md [hunk 1]
```diff
@@ -27,22 +27,22 @@ Estimated Time: **60 minutes**
 
 In this exercise, you will retrieve and display the list of sessions from a web service.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD05_LAB_MANUAL.md]: Should this be "...by using a web service"? If I understand this correctly, the web service brings the data "from" a database.

File: Instructions/20480C_MOD05_LAB_MANUAL.md [hunk 2]
```diff
@@ -51,139 +51,128 @@ Finally, you will run the application and view the Schedule page to verify that
...
+#### Task 2: Create the downloadSchedule function
 
 1.	In the **schedule.js** file, find the comment that starts with the following text:
-    ```javascript
+```javascript
         // TODO: Create a function called downloadSchedule
-    ```
-2.	Create an empty function named **downloadSchedule**. You will add code to this function in subsequent tasks to asynchronously get a list of sessions from a web service.
+```
+2.	Create an empty function named **downloadSchedule**. You will add code to this function in subsequent tasks to get a list of sessions from a web service asynchronously.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD05_LAB_MANUAL.md]: Should this be "...by using a web service"? If I understand this correctly, the web service brings the data "from" a database.

File: Instructions/20480C_MOD05_LAB_MANUAL.md [hunk 3]
```diff
@@ -51,139 +51,128 @@ Finally, you will run the application and view the Schedule page to verify that
...
- 
->**Note:** The URL **/schedule/list?fail** generates errors that enable you to test that the exception handling parts of your code work correctly.
+ >**Note**: The **/schedule/list?fail** URL generates errors that enable you to test that the exception handling parts of your code work correctly.
+4.	Run the application again, and then verify that the **Service currently unavailable** error message appears.
+5.	After testing, close Microsoft Edge, and then change the URL back to **/schedule/list**.
 
-End of RLO
-4.	Run the application again and verify that the error message **Service currently unavailable** appears.
-5.	After testing, close Microsoft Edge and change the URL back to **/schedule/list**.
+>**Results**: After completing this exercise, you will have modified the code for the **Schedule** page to display the list of sessions retrieved from a web service, and to handle errors that can occur when communicating with a remote service.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD05_LAB_MANUAL.md]: Should this be "...by using a web service"? If I understand this correctly, the web service brings the data "from" a database.

File: Instructions/20480C_MOD05_LAB_MANUAL.md [hunk 4]
```diff
@@ -51,139 +51,128 @@ Finally, you will run the application and view the Schedule page to verify that
...
-2.	Also, handle the exceptional case of the remote data source returning invalid JSON data that cannot be parsed. In this case, display a general error message using the **alert** function.
+2.	Also, handle the exceptional case of the remote data source returning invalid JSON data that cannot be parsed. In this case, use the **alert** function to display a general error message.
 
-#### Task 4: Test the Schedule page.
+#### Task 4: Test the Schedule page
 
-1.	Run the application and view the **schedule.htm** page to verify that the list of sessions is displayed.
+1.	Run the application, and then go to the **schedule.htm** page to verify that the list of sessions is displayed.
 2.	Close Microsoft Edge.
 3.	In the **schedule.js** file, change the requested URL to **/schedule/list?fail**, save the file, and then refresh the **schedule.htm** page in Microsoft Edge.
```
Comment (VinayApposite) [inline Instructions/20480C_MOD05_LAB_MANUAL.md]: We have asked the student to close Edge in the previous step. Should we move that down to after this step?

Comment (yogesh-golande) [reply Instructions/20480C_MOD05_LAB_MANUAL.md]: Might give cache or hot deployment sometime. So better to keep them separate and rerun the application after the change.

File: Instructions/20480C_MOD05_LAB_MANUAL.md (hunk 3 above)
Comment (yogesh-golande) [reply Instructions/20480C_MOD05_LAB_MANUAL.md]: Yes. You are right.

File: Instructions/20480C_MOD05_LAB_MANUAL.md (hunk 2 above)
Comment (yogesh-golande) [reply Instructions/20480C_MOD05_LAB_MANUAL.md]: Yes. You are right.

File: Instructions/20480C_MOD05_LAB_MANUAL.md (hunk 1 above)
Comment (yogesh-golande) [reply Instructions/20480C_MOD05_LAB_MANUAL.md]: Yes. You are right.

**Pull request merged**